.ci.yaml

```diff
@@ -236,29 +236,33 @@ targets:
 
   - name: Linux linux_arm_host_engine
     recipe: engine_v2/engine_v2
-    timeout: 60
```

Comment:
This will increase the timeouts for main which may not be what we are looking for.

Reply:
Is this this the job thats timing out though at 1 hour: https://ci.chromium.org/ui/p/dart-internal/builders/flutter/Linux%20Production%20Engine%20Drone/2560/overview

Reply:
Or, does the higher level job's timeout not kick in if its waiting on sub tasks?

Reply:
Not sure if that is implemented, let me take a look.

Reply:
I have a prototype change with increased build timeouts I'm testing right now, will put up a draft if it works.